benchmarks/build.gradle

```diff
@@ -13,15 +13,18 @@ configurations {
 dependencies {
   // Use the baseline to avoid using new APIs in the benchmarks
   compileOnly libs.reactor.perfBaseline.core
+  compileOnly libs.reactor.perfBaseline.coreMicrometer
   compileOnly libs.jsr305
 
   implementation "org.openjdk.jmh:jmh-core:$jmhVersion"
   implementation libs.reactor.perfBaseline.extra, {
     exclude group: 'io.projectreactor', module: 'reactor-core'
   }
+  implementation platform(libs.micrometer.bom)
   annotationProcessor "org.openjdk.jmh:jmh-generator-annprocess:$jmhVersion"
 
   current project(':reactor-core')
+  current project(':reactor-core-micrometer')
```

Comment:
Please also modify the baseline source set and add the reactor-core-micrometer module with appropriate version. The idea here is that we can have a comparison of performance in the previously released version against the current version. That's why there's a "current" and a "baseline" source set.